recipes/deepspeech/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "deepspeech" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mozilla/DeepSpeech/archive/v{{ version }}.tar.gz
+  sha256: a8e731a27a7cac4b4d0ae72b8e91493a5a372888dca10a11ece65ce16fb9b1b2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - absl-py
+    - attrdict
+    - bs4
+    - numpy
+    - optuna
+    - opuslib
+    - pandas
+    - progressbar2
+    - pyogg
+    - pyxdg
+    - resampy
```

Should probably match the versions listed upstream, since conda-forge has some that don't match:
```suggestion
    - opuslib ==2.0.0
    - pandas
    - progressbar2
    - pyogg
    - pyxdg
    - resampy >=0.2.2
```